modules/cli-installing-cli-rpm.adoc

```diff
@@ -41,11 +41,20 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
 # subscription-manager attach --pool=<pool_id>
 ----
 
-. Enable the repositories required by {product-title} {product-version}:
+. Enable the repositories required by {product-title} {product-version}.
+
+** For Red Hat Enterprise Linux 8:
```

Comment:
Is it appropriate to use `{op-system-base}` or `{op-system-base-full}` here and in the below step instead of stating `Red Hat Enterprise Linux`?

Reply:
Not in this case because the RPMs used are RHEL-specific (not for CoreOS). This module is not included for OKD.